.github/workflows/test_generated_app.yml

```
@@ -13,6 +13,10 @@ env:
   NODE_VERSION: 16
   RAILS_VERSION: 7.0.1
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
```

Why do we need to skip it in the `main` branch? Isn't it better to cancel the test when we merge multiple PRs at the same time?
Should it just be:

```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```